backend/app/Models/BatchStepOutputValue.php

```
@@ -0,0 +1,85 @@
+use App\Models\Concerns\SoftDeletesWithAudit;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+/**
+ * The value an operator recorded for a typed step output on a specific batch
+ * step. Exactly one typed column is set per the output's value_type; a `picture`
+ * output stores the file on the private disk. A live row = recorded; clearing
+ * soft-deletes it (audit preserved) so it can be re-recorded.
+ */
+class BatchStepOutputValue extends Model
```

PHPStan (2.2.7)

[error] Class App\Models\BatchStepOutputValue extends unknown class Illuminate\Database\Eloquent\Model.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)